packages/apollo/lib/constants/index.ts

```diff
@@ -0,0 +1 @@
+export * from './apollo.constants';
```

Comment:
Not sure why we need this change but since you've already moved that file - let's keep it as is